cfme/automate/import_export.py

```diff
@@ -83,7 +84,7 @@ def import_domain_from(self, branch=None, tag=None):
         git_select = self.create_view(GitImportSelectorView)
         assert git_select.is_displayed
         git_select.flash.assert_no_error()
-        assert git_select.fill(self.fill_values_branch_select(branch, tag))
```

Comment:
Doesn't make sense if want to import from origin/master.

Reply:
Is this **REQUIRED**?

Reply:
For this test is not. I found this issue when I tried to import a domain from master branch.